README.md

```diff
@@ -32,12 +38,25 @@ aws cloudformation deploy \
 TODO: Add example of exporting an existing network
 
 ### Infrastructure stack
+
+__Parameters:__
++ NetworkStackName
+  + Description: The name of the parent networking stack created.
+  + Default: "mellon-network"
++ EnvironmentName
+  + Description: Any value that describes where the service exists
+  + Default: "dev"
++ DomainName
+  + Description: 'The DomainName to be used for all entities to be created. All services will be built within this Domain'
+  + Default: 'library.nd.edu'
```

Comment:
This is the type of thing that needs to be made super generic for community adoption: try `example.domain.com`. I also am not sure we even want to have this section in the README - it depends on if there are default values already specified in the CloudFormation(s) yet.